dbt/runner.py

```
@@ -239,6 +239,9 @@ def execute_model(profile, model, existing):
 
     result = None
 
+    if get_materialization(model) == 'ephemeral':
+        return 'SKIP'
```

we're just going to silently skip these, right?